Code/Tools/ProjectManager/Platform/Mac/ProjectUtils_mac.cpp

```diff
@@ -61,5 +62,37 @@ namespace O3DE::ProjectManager
 
             return AZ::Success(xcodeBuilderVersionNumber);
         }       
+
+        AZ::Outcome<void, QString> OpenCMakeGUI(const QString& projectPath)
+        {
+            const QString cmakeHelp = QObject::tr("Please verify you've installed CMake.app from " 
+                            "<a href=\"https://cmake.org\">cmake.org</a> or, if using HomeBrew, "
+                            "have installed it with <pre>brew install --cask cmake</pre>");
+            QString cmakeAppPath = QStandardPaths::locate(QStandardPaths::ApplicationsLocation, "CMake.app", QStandardPaths::LocateDirectory);
+            if (cmakeAppPath.isEmpty())
+            {
+                return AZ::Failure(QObject::tr("CMake.app not found. ") + cmakeHelp);
+            }
+
+            QString projectBuildPath = QDir(projectPath).filePath(ProjectBuildPathPostfix);
+            AZ::Outcome result = GetProjectBuildPath(projectPath);
+            if (result.IsSuccess())
+            {
+                projectBuildPath = result.GetValue();
+            }
+
+            QProcess process;
+
+            // if the project build path is relative, it should be relative to the project path 
+            process.setWorkingDirectory(projectPath);
+            process.setProgram("open");
+            process.setArguments({ "-a","CMake","--args","-S", projectPath, "-B", projectBuildPath });
+            if(!process.startDetached())
+            {
+                return AZ::Failure(QObject::tr("CMake.app failed to open. ") + cmakeHelp);
```

Comment:
cosmetic: is the space at the end intentional? noticed the same on line 74, but not on `ProjectUtils_windows.cpp` line 99